hdk/assembly/code.py

```diff
@@ -200,3 +202,23 @@ def translate(instructions: Iterable[Instruction]) -> Iterator[str]:
             yield translate_a_instruction(instruction, symbol_table)
         else:
             yield translate_c_instruction(instruction)
+
+
+def link_instructions(instructions: Iterable[Instruction]) -> list[Command]:
+    """Translates instructions in commands for bringing variables into the table.
+
+    Args:
+        instructions: An iterable of symbolic instructions.
+
+    Returns: An list of commands
```

Comment:
Move "An list of commands to the next line", see other docstrings with Returns. Improve English.